dockertest/subtest.py

```diff
@@ -377,7 +381,8 @@ def initialize(self):
             if len(children) < 1:
                 return  # No children specified
             # Any children NOT listed in fullnames are 'undefined'
-            if children - fullnames:
+            # unless they were excluded by (an old) bugzilla
+            if children - fullnames - bzexclude:
                 msg = ("Sub-subtest(s) %s referenced in control include, "
                        "exclude, and/or subthings but not defined in "
                        "subsubtests configuration option: %s"
```

Comment:
The line right below this, shouldn't it be `% (children i fullnames - bzexclude` ? And if so, would it make sense to refactor that long list into something like

    foo = children - fullnames - bzexclude
    if foo: